_gettingstarted/4-python-highcharts-core.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: article
+title: Highcharts-core
+description: A comprehensive guide on how to work with Highchats-core, a Python library for creating dynamic charts.
+tags: [highcharts-core, charts, python, pip]
+image: https://learn.piptrends.com/assets/images/getting-started/getting-started-python-docs-cover.png
```

Comment:
You can use this figma file to make the cover - https://www.figma.com/file/lH5XXCqo5DTwDOc4Pb7m2S/pip-Trends-assests?type=design&node-id=1%3A2&mode=design&t=5uwZ7VHQJKwIcXao-1

Reply:
Even the image - cover can be changed